server/middleware/auth.go

```diff
@@ -162,13 +166,18 @@ func authFunction(ctx context.Context, jwtValidators []*validator.Validator, con
 		return ctx, err
 	}
 
-	validatedToken, found := cache.Get(tkn)
-	log.Debug().Bool("validated token found", found).Interface("token", validatedToken)
-	if !found {
+	var validatedToken interface{}
+	if !BypassAuthCaches(ctx) {
+		validatedToken, err = cache.Get(tkn)
+		if err != nil {
```

Comment:
if an item doesn't exist then does cache throws an error? Or only in case of legit errors?

Reply:
Yes it returns an error if item doesn't exists in cache. I refactored this code as per your above comment and this is gone now.